node/src/chain_spec.rs

```diff
@@ -352,7 +352,7 @@ fn centrifuge_genesis(
 			"chainId": Into::<u32>::into(chain_id),
 		},
 		"evm": {
-			"accounts": runtime_common::evm::precompile::utils::precompile_account_genesis::<centrifuge_runtime::Precompiles>(),
+			"accounts": runtime_common::evm::utils::account_genesis::<centrifuge_runtime::Precompiles>(),
```

Comment:
Do we also change this for testing purposes right?

Reply:
Yes.

Reply:
@wischli we should probably also generate a new dev genesis, right? For the life environment, or don't we use a static one?